app/commands.py

```diff
@@ -1227,18 +1227,13 @@ async def server(ctx: Context) -> str | None:
     uptime = int(time.time() - proc.create_time())
 
     # get info about our cpu
-    with open("/proc/cpuinfo") as f:
-        header = "model name\t: "
-        trailer = "\n"
-
-        model_names = Counter(
-            line[len(header) : -len(trailer)]
-            for line in f.readlines()
-            if line.startswith("model name")
-        )
+    cpu_info = cpuinfo.get_cpu_info()
 
     # list of all cpus installed with thread count
-    cpus_info = " | ".join(f"{v}x {k}" for k, v in model_names.most_common())
+    thread_count = cpu_info["count"]
+    cpu_name = cpu_info["brand_raw"]
+
+    cpu_info_str = f"{thread_count}x {cpu_name}"
```

Comment:
can we use shutil?

Reply:
if you're prepared to write parsers for the n amount of different os specific commands for this be my guest

it's just not worth it 😆

Reply:
oops sorry i meant psutil, not shutil..

i think it has multiplatform support and we already have it installed

Reply:
it can be used to provide usage information but not threads / name

if you want to display cpu usage instead then sure, i just wanted to mimic previous behaviour

Reply:
we can get thread count with https://psutil.readthedocs.io/en/latest/#psutil.cpu_count

i'm not sure how to do cpu name though.. we can get architecture & processor using https://docs.python.org/3/library/platform.html but idk how py-cpuinfo is doing the full model name